lib/iris/cube.py

```diff
@@ -716,9 +714,8 @@ def __init__(self, data, standard_name=None, long_name=None,
         if isinstance(data, six.string_types):
             raise TypeError('Invalid data type: {!r}.'.format(data))
 
-        self._dask_array = None
-        self._numpy_array = None
-        self.data = data
+        # Initialise the cube data manager.
+        self._data_manager = DataManager(data, realised_dtype=dtype)
```

Comment:
There is unfinished business here (though not the fault of this PR):
The new kwargs `fill_value` and `dtype` need coverage in the `__init__` docstring.
That might have made this clearer!